### ISSUE TITLE
CxxXunitSensor doesn't run, not sure what I'm doing wrong.

### ISSUE THREAD

**Issue by Vishrs** [SonarOpenCommunity/sonar-cxx#1063]
Using sonar-cxx-plugin-0.9 and SonarQube Runner 2.4 with SonarQube server 4.5.7. I have a sonar runner properties file in my workspace with 

> sonar.cxx.xunit.reportPath=ut_results/*_ut.xml
 
I'm using Google test framework for my unit tests and using its xmloutput option to general the xml files.

Everything else works and here is the sensors running:
`10:21:32.473 DEBUG - Sensors : QProfileSensor -> SquidSensorCpp -> InitialOpenIssuesSensor -> ProjectLinksSensor -> VersionEventsSensor -> FileHashSensor -> Cppcheck -> CPD Sensor (wrapped)`

What could be wrong?

**Comment by guwirth** [comment]
@Vishrs please turn debug logging on, maybe you get more information: https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Get-debug-information.
By the way: Why are you not using latest LTS 5.6?

**Comment by Vishrs** [comment]
Sonar Runner is being executed by the Jenkins server I believe. I'm not sure if I have control over this as I'm new to this environment, but I will ask about this. Its currrently running with the -e option, i'll also try the -X.

Thanks

**Comment by guwirth** [comment]
> sonar.cxx.xunit.reportPath=ut_results/*_ut.xml

Please read also this: https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Path-and-path-separator-issues

**Comment by Vishrs** [comment]
It turns out that sonar is not actually using the c++ community plugin but I believe it is still using the commercial plugin. I have removed the plugin from my extensions/plugins directory and restarted the sonar server. I have also removed any references to the commercial SonarC++ plugin properties from my sonar-project.properties file. It is still not using sonar-cxx-plugin. Is there anything else I'm not thinking of?
